src/tests/elementary/meson.build

```diff
@@ -174,6 +176,10 @@ efl_ui_suite_src = [
   'custom_recognizer2.c',
 ]
 
+if get_option('dbus')
+  efl_ui_suite_src += ['efl_ui_test_atspi.c']
```

Comment:
```suggestion
  efl_ui_suite_src += files('efl_ui_test_atspi.c')
```

Reply:
`efl_ui_suite_src` don't use files, it is only a list of names